deploy/charts/rook-ceph/templates/deployment.yaml

```diff
@@ -85,7 +85,12 @@ spec:
 {{- end }}
         - name: ROOK_DISABLE_DEVICE_HOTPLUG
           value: "{{ .Values.disableDeviceHotplug }}"
-
+        - name: DISCOVER_DAEMON_UDEV_BLACKLIST
```

Comment:
These settings were moved to configmap.yaml by #12840. Let's keep them there.

Reply:
Indeed, missed that when I rebased